tests/Unit/Execution/Arguments/ArgumentSetTest.php

```diff
@@ -29,6 +29,27 @@ public function testHasArgument(): void
         $this->assertTrue($set->has('foo'));
     }
 
+    public function testExistsArgument(): void
```

Comment:
```suggestion
    public function testExists(): void
```

Let's also rename the other test to `testHas`.